Makefile

```diff
@@ -116,11 +116,15 @@ help:  ## Display this help
 
 .PHONY: test
 test: ## Run tests.
+	source ./scripts/fetch_ext_bins.sh; fetch_tools; setup_envs; go test ./... $(TEST_ARGS)
+
+.PHONY: test-verbose
+test-verbose: ## Run tests with verbose settings.
```

Comment:
Question: do we need an additional test target given that this could be achieved via TEST_ARGS="-v"? might be it will be enough to document this in https://cluster-api.sigs.k8s.io/developer/testing.html#running-unit-and-integration-tests...

Reply:
See https://github.com/kubernetes-sigs/cluster-api/issues/4182#issuecomment-777703195.

Reply:
Ohh, thanks, nevermind then